### DISCUSSION TITLE
Проанализированные статьи

### DISCUSSION THREAD

**Discussion by almkuznetsov** [almkuznetsov/bmstudemo#38] [General]
Поскольку мы изучаем много статей, но не все из них используем в базе, будет полезно иметь место, куда можно записать все так или иначе проанализированные статьи с кратким описанием. Пока не придумал ничего лучше, чем писать их здесь.
Формат предлагаю следующий:
Название статьи (полное по ГОСТу с scholar.google.com, например) - краткое описание в свободной форме

**Comment by nedomolkinaeyu** [comment]
Карпов А. В., Лепешкин О. М. Моделирование технических каналов утечки информации на распределенных объектах управления //International Journal of Advanced Studies. – 2018. – Т. 8. – №. 1. – С. 69-83. 
О количественной оценке риска утечки информации, необходимой для обоснования рационального состава организационно-технических мер защиты, а также варианта структуры ЗИ от УТК.

**Comment by nedomolkinaeyu** [comment]
Зеленский А. А., Короченцев Д. А., Ревякина Е. А. Разработка адаптивного fuzzy-алгоритма идентификации технических каналов утечки информации //Colloquium-journal. – Голопристанський міськрайонний центр зайнятості, 2020. – №. 11 (63). – С. 20-24.
Описана модель обеспечения информационной безопасности, в основу которой положена адаптивная нейро-нечёткая система.

**Comment by nedomolkinaeyu** [comment]
Привалов А. А., Евглевская Н. В. Математическая модель вскрытия нарушителем технических каналов утечки информации //Бюллетень результатов научных исследований. – 2014. – №. 3 (12). – С. 102-109.
Здесь поставлена задача и указано её решение с помощью процесса в виде стохастической сети.

**Comment by nedomolkinaeyu** [comment]
Назаренко К. Н. Функциональные модели для обоснования требований к нормативному обеспечению мероприятий по выявлению технических каналов утечки информации //Вестник Воронежского института МВД России. – 2007. – №. 2. – С. 221-225.
В результате анализа стратегий совершения преступлений в информационной сфере установлено, что противоправные действия можно описать концептуально в виде шестиэтапной функциональной модели.

**Comment by nedomolkinaeyu** [comment]
Аштаева Н. В. Исследование технических каналов утечки информации //Тенденции развития науки и образования. – 2021. – №. 69-1. – С. 6-9.
Описание каналов (у нас есть вся информация)
